src/object/schema_system_catalog_install.cpp

```diff
@@ -818,7 +819,9 @@ namespace cubschema
       {"type_name", format_varchar (16)}
     },
 // constraints
-    {},
+    {
+      {DB_CONSTRAINT_PRIMARY_KEY, "", {"type_id", nullptr}, false}
```

Comment:
```suggestion
      {DB_CONSTRAINT_PRIMARY_KEY, "", {"type_id","type_name", nullptr}, false}
```

db_attribute 카탈로그 뷰를 조회할 때 covered 인덱스를 활용하려면 (type_id, type_name) 순서로 복합 PK를 생성하는 것이 효과적이며, 이 경우 인덱스는 covered: true로 동작합니다.
```
Trace Statistics:
  SELECT (time: 79, fetch: 19587, fetch_time: 19, ioread: 17)
    SCAN (table: _db_class), (heap time: 2, fetch: 244, ioread: 1, readrows: 59, rows: 59)
      SCAN (index: _db_attribute.i__db_attribute_class_of_attr_name_attr_type), (btree time: 18, fetch: 7740, ioread: 6, readkeys: 454, filteredkeys: 0, rows: 454) (lookup time: 3, rows: 454)
        SCAN (index: _db_domain.i__db_domain_object_of), (btree time: 17, fetch: 9080, ioread: 3, readkeys: 454, filteredkeys: 0, rows: 454) (lookup time: 3, rows: 454)
          **SCAN (index: _db_data_type.pk__db_data_type_type_id_type_name), (btree time: 12, fetch: 1816, ioread: 0, readkeys: 454, filteredkeys: 0, rows: 454, covered: true)**
  ...
```

Reply:
감사합니다.
반영하겠습니다.